apim-operator/deploy/controller-configs/mgw_conf_mustache.yaml

```diff
@@ -127,4 +127,4 @@ data:
         password = "root"
 
     [httpClients]
-      verifyHostname={{verifyHostname}}
\ No newline at end of file
+      verifyHostname=false
```

Comment:
This is not the way to do this. Let's set the configmap value to false. Not the template

Reply:
Fixed